learning_observer/learning_observer/canvas.py

```diff
@@ -0,0 +1,67 @@
+import functools
+
+import learning_observer.log_event
```

Comment:
Unused imports. Please run this through `pylint` and `pycodestyle` to find issues like this. Also, imports should be correctly ordered (unless there is good reason otherwise: system, package, local, and alphabetically within each)

Reply:
When running through the linters, make sure to use the same settings as the rest of the system. See the Makefile listing-python section for details.

Reply:
I applied linters to the code and cleaned up based on the comments